src/browser/pages/vscode.ts

```diff
@@ -95,8 +95,64 @@ try {
   console.error(error)
 }
 
+export function setBodyBackgroundToThemeBackgroundColor(document: Document, localStorage: Storage) {
```

Comment:
At first, I didn't like the function signature — passing in `document` and `localStorage` — but then I justified it because:
- this function is only called in one place 
- it makes it _a lot_ easier to test when `document` or `localStorage` is missing

Reply:
makes sense to me, this also allows you to test this by mocking document/localStorage, and avoids global state